samples/dotnet/kernel-syntax-examples/Example12_Planning.cs

```diff
@@ -19,10 +19,50 @@ internal static class Example12_Planning
 {
     public static async Task RunAsync()
     {
-        await PoetrySamplesAsync();
-        await EmailSamplesAsync();
-        await BookSamplesAsync();
-        await MemorySampleAsync();
+        //await PoetrySamplesAsync();
+        //await EmailSamplesAsync();
+        //await BookSamplesAsync();
+        //await MemorySampleAsync();
+        await ConditionalSampleAsync();
+    }
+
+    private static async Task ConditionalSampleAsync()
+    {
```

Comment:
Can you also add an integration test or two so when we refactor away from Planning Skill we can make sure things don't break easily.

Reply:
Definitely was talking about it yesterday, will add some integration tests for the supported scenarios.

Reply:
Integration tests added.